src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/DotNetSdkHelper.cs

```diff
@@ -180,7 +180,10 @@ void processConfigCallback(Process process)
             {
                 if (e.Data?.Contains("Application started. Press Ctrl+C to shut down.") ?? false)
                 {
-                    ExecuteHelper.ExecuteProcessValidateExitCode("kill", $"-s TERM {process.Id}", OutputHelper);
+                    [DllImport("Kernel32.dll")]
```

Comment:
Failing cause of this. This was working at line 211 so I don't understand why it's failing now.

Unhandled exception. System.DllNotFoundException: Unable to load shared library 'Kernel32.dll' or one of its dependencies. In order to help diagnose loading problems, consider using a tool like strace. If you're using glibc, consider setting the LD_DEBUG environment variable: 
  /home/vsts/work/1/s/.dotnet/shared/Microsoft.NETCore.App/8.0.0-rc.2.23479.6/Kernel32.dll.so: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/artifacts/bin/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/Release/net8.0/Kernel32.dll.so: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/.dotnet/shared/Microsoft.NETCore.App/8.0.0-rc.2.23479.6/libKernel32.dll.so: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/artifacts/bin/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/Release/net8.0/libKernel32.dll.so: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/.dotnet/shared/Microsoft.NETCore.App/8.0.0-rc.2.23479.6/Kernel32.dll: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/artifacts/bin/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/Release/net8.0/Kernel32.dll: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/.dotnet/shared/Microsoft.NETCore.App/8.0.0-rc.2.23479.6/libKernel32.dll: cannot open shared object file: No such file or directory
  /home/vsts/work/1/s/artifacts/bin/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/Release/net8.0/libKernel32.dll: cannot open shared object file: No such file or directory

Reply:
Failing on Mac/Linux. WIndows is fine

Reply:
Fixed with Preprocessor Directives and adding back in the ExecuteHelper line. That apparently works on OSX and Linux but not windows, while DllImport only works for windows.